toplevel/ccompile.ml

```diff
@@ -145,18 +149,20 @@ let compile opts copts ~echo ~f_in ~f_out =
       let _doc = Stm.join ~doc:state.doc in
       let wall_clock2 = Unix.gettimeofday () in
       check_pending_proofs ();
-      if mode <> BuildVok  (* Don't output proofs in -vok mode *)
-        then Library.save_library_to ~output_native_objects Library.ProofsTodoNone ldir long_f_dot_out (Global.opaque_tables ());
+      (* In .vo production, dump a complete .vo file.
+         In .vok production, only dump an empty .vok file. *)
+      if mode <> BuildVok
```

Comment:
This is a nit, but in case you amend commits: I'd write `if mode = BuildVok then empty_file else save_library`

Reply:
(as you do below when `mode = BuildVo`)